README.md

```diff
@@ -23,7 +23,7 @@ with the chip on your system and start the emulated device.
 1. AS A NONROOT USER: In the `rust/` directory, run `./build.sh`.
    This step will take a long time.
 2. From the root of the project, run `yarn install`
```

Comment:
could just be `yarn` instead of `yarn install` - that's how the cool kids do it AFAIK